src/frontend/src/components/common/Modal/_modal.scss

```diff
@@ -5,17 +5,47 @@
   width: 100%;
   height: 100%;
   background: rgba(0, 0, 0, 0.6);
+  border-radius: 5px;
 
+  .modal-title {
+    display: block;
+    font-weight: bold;
+    position: relative;
```

Comment:
`position: relative` is used when you want to use `position: absolute` on a child element. Unless I am missing it, there isn't anything being absolutely positioned within this element.

Reply:
remove